DirectGuiExtension/DirectTooltip.py

```diff
@@ -93,15 +93,18 @@ def updateTooltipPos(self, task):
             if self.getParent() == base.pixel2d:
                 x = base.win.getPointer(0).getX()
                 y = -base.win.getPointer(0).getY()
+                # set the text to the current mouse position
+                self.setPos(x + self.textXShift,
+                            0,
+                            y + self.textYShift)
             else:
                 x = (base.mouseWatcherNode.getMouseX()*aspX)
                 y = (base.mouseWatcherNode.getMouseY()*aspY)
-
-            # set the text to the current mouse position
-            self.setPos(
-                x + self.textXShift,
-                0,
-                y + self.textYShift)
+                # set the text to the current mouse position
+                self.setPos(base.aspect2d,
```

Comment:
I think instead of base.aspect2d this should rather be `self.getParent()` to also work correctly if we have another parent.

Reply:
Are you sure? Since the mouse coordinates that get calculated are relative to aspect2d , shouldn't we also set the position relative to aspect2d? When I try and add the toolTip in DirectGuiDesigner that was needed to make it move correctly.

Reply:
Yes, sorry, I missed that we shift relative to mouse position. Should be fine then.